test/tests/parallel/set2/test_helper.bats

```diff
@@ -130,43 +140,55 @@ load 'test_helper/common'
   assert_failure
 }
 
-@test "container_has_service_running/wait_for_service" {
+@test "${TEST_NAME_PREFIX} container_has_service_running/wait_for_service" {
+  # the container name is used here explicitly, because it cannot be a local
+  # variable as the `teardown` function would not have the correct scope
+  # and does not see the variable
```

Comment:
Same feedback here regarding `teardown()` shouldn't require usage elsewhere to avoid referencing a var.

Reply:
I did not put these in place, but cleaned up some of them as the notice is confusing I think.

Reply:
This is now reverted.